platform/ui/src/utils/ModalProvider.js

```diff
@@ -65,6 +82,9 @@ const ModalProvider = ({ children, modal: Modal }) => {
 ModalProvider.propTypes = {
   children: PropTypes.node,
   modal: PropTypes.node,
+  service: PropTypes.shape({
+    setServiceImplementation: PropTypes.func,
```

Comment:
if service is defined, then this property of it's shape can be required

Reply:
Done! thank you.